src/main/java/github/scarsz/discordsrv/listeners/PlayerAdvancementDoneListener.java

```diff
@@ -59,21 +64,46 @@ public void onPlayerAdvancementDone(PlayerAdvancementDoneEvent event) {
             return;
         }
 
+        TextChannel channel = DiscordSRV.getPlugin().getMainTextChannel();
+        String channelName = channel.getName();
```

Comment:
This should be the in-game channel, not Discord.

Reply:
The previous code used `DiscordSRV.getPlugin().getMainTextChannel()` am I not using the same thing here? If not what method would you prefer I got the channel using?

Reply:
getMainChatChannel()